tools/harness-simulation/harness/Web/data/deviceInputFields.xml

```diff
@@ -0,0 +1,11 @@
+<DEVICE_FIELDS>
+    <DEVICE name="OpenThread_Sim" thumbnail="OpenThread.png" description = "OpenThread Simulation" THCI="OpenThread_Sim">
+        <Hardware thumbnail="POSIX.png" description="OpenThread Simulation FTD"></Hardware>
```

Comment:
nit: Maybe it's not very accurate to say a simulation device is a `POSIX` device. It might be confused with a POSIX FTD.

Reply:
Any idea of a proper hardware thumbnail? The device thumbnail currently is the same as other OpenThread real devices. However, simulations do not "own" a hardware. Perhaps it is better to remove the hardware field, just leaving the hardware thumbnail a blank.

Reply:
Currently the hardware thumbnail is removed.

Reply:
It's better if we are using a simulation thumbnail. 

I think we can use https://avatars.githubusercontent.com/u/16142269?s=280&v=4 for now.